CONTRIBUTING.md

```diff
@@ -91,15 +91,54 @@ it will create a new folder inside `packages` with a simple boilerplate for your
 
 ### Prerelease
 
-- Every merged PR should trigger an automatic canary release
+- Every commit or PR merged to the master branch will trigger the Github [prerelease](https://github.com/Turfjs/turf/blob/master/.github/workflows/prerelease.yml) action, creating an alpha release (e.g. `7.0.0-alpha.116` where 116 is the number of commits to master since the last release tag).
 - The version number is calculated by a combination of the output of `git describe` and the `publish:prerelease` script in the root package.json. Specifically having `major` is appropriate for the pre-7.0 development phase, but we should change it to `minor` after 7.0.0 is finally released.
 
 ### Release
-- If necessary, make and merge a PR to contain any last minute changelog entries or other housekeeping items
-- Make a PR that contains the file changes from `pnpm lerna version --no-push 7.0.0` (update the version as necessary)
-- When pushing the release PR, use `--follow-tags` to make sure the relevant tag is also pushed
-- The CI will start doing the release when the tag is created
-- Don't forget to merge your PR
+- If necessary, make and merge a PR with any last minute housekeeping items (docs, etc.)
+- Review code commits and decide the new version number.  If there are breaking changes, then it should be a major version bump, e.g. 6.x.x to 7.0.0.  This project follows [semantic versioning](https://semver.org/).
+
+#### Release commands
+Run the following release commands, replacing 7.0.0 with the version number to release:
+
+- fetch the latest code from remote origin
+```bash
+git fetch origin
+```
+
+- create a release branch, replace mf with your initials to make it clear whose branch it is.
+```bash
+get checkout origin/master -b mf/release-7.0.0
+```
+
+- increment the version number of all packages, without pushing to origin.  This will also create a release tag.
+```bash
+pnpm lerna version --no-push 7.0.0
+```
+
+- check the changed files and stage them for commit.  Example:
+```bash
+git add packages/*/package.json
```

Comment:
lerna.json also gets updated right? Should that be committed at this stage as well @mfedderly?

Reply:
I was writing these instructions from memory so I probably just forgot that it changed, but yes everything the lerna command changes needs to get added.

Reply:
I've simplified the add command to stage all modifications.

Reply:
Actually I switched it to being more specific, avoid any surprises 🤷 

```bash
git add packages/*/package.json
git add lerna.json
```